hackclub.com.yaml

```diff
@@ -1728,11 +1733,18 @@ knit:
 kr:
   ttl: 3600
   type: CNAME
-  value: cname.vercel-dns.com.
+  value: counterspell-busan.netlify.app.
```

Comment:
This is the counterspell Busan site, not the site of your Hack Club in Korea.

Reply:
Also -- ISO 3166 codes should be reserved for hackclub/global

Reply:
Indeed, are you fine with just the .korea subdomain @juni37?

Reply:
@Muirrum @alx-alexpark It seems like hackclub/global is not maintained at the moment and redirects to the main hackclub.com domain. I am fine with korea.hackclub but could you double check if the repository is currently maintained and enforced?

Reply:
2-letter country codes are currently reserved for future use, and yes hackclub/global is still maintained.

Reply:
understood! thanks.